client/blocks/login/login-form.jsx

```diff
@@ -309,13 +311,16 @@ export class LoginForm extends Component {
 
 						<label htmlFor="usernameOrEmail">
 							{ this.isPasswordView() ? (
-								<a href="#" className="login__form-change-username" onClick={ this.resetView }>
+								<button
```

Comment:
Is it possible to use our <Button /> component?

Reply:
I'm not very happy with how our `Button` supports the `is-link` option. The styles are slightly broken and it's just easier to create a plain `button` element with a little CSS to set the text and hover colors.

I have some ideas how to:
1. improve the "CSS reset" styles that convert the native `button` to an unstyled div-like element. That's important for a lot of a11y cases where we want to use `button` for generic clickable area that natively reacts to keyboard events, too.
2. introduce a properly debugged `<Button link />` component.

But that's for another PR. Thanks for bringing this up 🙂

Reply:
Another question: while tabbing through the form I found that the focus ring isn't there (probably because it's not a link anymore). Should we hold the styles to what button links currently look like on focus?

master:
<img width="194" alt="Screenshot 2019-06-27 at 15 15 28" src="https://user-images.githubusercontent.com/9202899/60269483-d41fc100-98ee-11e9-9ecc-2700f883e983.png">

this branch:
<img width="168" alt="Screenshot 2019-06-27 at 15 15 35" src="https://user-images.githubusercontent.com/9202899/60269485-d41fc100-98ee-11e9-8360-6884d945628f.png">

`<Button />`:
<img width="140" alt="Screenshot 2019-06-27 at 15 16 48" src="https://user-images.githubusercontent.com/9202899/60269486-d41fc100-98ee-11e9-90c5-754402b0c1eb.png">